setup.py

```diff
@@ -14,4 +19,4 @@
       description='Cloudstate Python Support Library',
       packages=find_packages(exclude=['tests', 'shoppingcart']),
       long_description=open('README.md').read(),
-      zip_safe=False)
\ No newline at end of file
+      zip_safe=False)
```

Comment:
note: I see many complaints by my IDE (PyCharm) about missing newlines on the last line (PEP 8: W292 no newline at end of file), so corrected this here. This is needed or good practice by other languages or filetypes, we might adhere to it too.